Mathlib/Probability/Distributions/Exponential.lean

```diff
@@ -11,163 +11,116 @@ import Mathlib.Probability.Distributions.Gamma
-  rw [lintegral_eq_lintegral_Ici_add_Iio (exponentialPdf r) 0, ← ENNReal.toReal_eq_one_iff]
-  have leftSide : ∫⁻ x in Iio 0, exponentialPdf r x = 0 := by
-    rw [set_lintegral_congr_fun measurableSet_Iio
-      (ae_of_all _ (fun x (hx : x < 0) ↦ exponentialPdf_of_neg hx)), lintegral_zero]
-  have rightSide :
-      ∫⁻ x in Ici 0, exponentialPdf r x = ∫⁻ x in Ici 0, ENNReal.ofReal (r * rexp (-(r * x))) :=
-    set_lintegral_congr_fun isClosed_Ici.measurableSet
-      (ae_of_all _ (fun x (hx : 0 ≤ x) ↦ exponentialPdf_of_nonneg hx))
-  simp only [leftSide, add_zero]
-  rw [rightSide, ENNReal.toReal_eq_one_iff, ← ENNReal.toReal_eq_one_iff]
-  rw [← integral_eq_lintegral_of_nonneg_ae (ae_of_all _ (fun _ ↦ by positivity))]
-  · have integrOn : IntegrableOn (fun x ↦ rexp (-(r * x))) (Ioi 0) := by
-      simp only [← neg_mul, exp_neg_integrableOn_Ioi 0 hr]
-    rw [integral_mul_left, integral_Ici_eq_integral_Ioi,
-        integral_Ioi_of_hasDerivAt_of_tendsto' (m := 0) (fun _ _ ↦ hasDerivAt_exp_neg hr) integrOn]
-    · field_simp
-    · rw [← mul_zero (-1/r)]
-      exact tendsto_const_nhds.mul (tendsto_exp_neg_atTop_nhds_0.comp
-        ((tendsto_const_mul_atTop_of_pos hr).2 tendsto_id))
-  · exact ((measurable_id'.const_mul r).neg.exp.const_mul r).stronglyMeasurable.aestronglyMeasurable
+lemma lintegral_exponentialPdf_eq_one {r : ℝ} (hr : 0 < r) : ∫⁻ x, exponentialPdf r x = 1 :=
+  lintegral_gammaPdf_eq_one zero_lt_one hr
 
 end ExponentialPdf
 
 open MeasureTheory
 
-/-- Measure defined by the exponential Distribution -/
+/-- Measure defined by the exponential distribution -/
 noncomputable
-def expMeasure (r : ℝ) : Measure ℝ :=
-  volume.withDensity (exponentialPdf r)
+def expMeasure (r : ℝ) : Measure ℝ := gammaMeasure 1 r
 
-instance instIsProbabilityMeasureExponential (r : ℝ) [Fact (0 < r)] :
-    IsProbabilityMeasure (expMeasure r) where
-  measure_univ := by simp [expMeasure, lintegral_exponentialPdf_eq_one r Fact.out]
+lemma isProbabilityMeasureExponential {r : ℝ} (hr : 0 < r) :
+    IsProbabilityMeasure (expMeasure r) := isProbabilityMeasureGamma zero_lt_one hr
 
 section ExponentialCdf
 
-/-- CDF of the exponential Distribution -/
+/-- CDF of the exponential distribution -/
 noncomputable
 def exponentialCdfReal (r : ℝ) : StieltjesFunction :=
   cdf (expMeasure r)
 
-lemma exponentialCdfReal_eq_integral (r x : ℝ) [Fact (0 < r)] :
+lemma exponentialCdfReal_eq_integral {r : ℝ} (hr : 0 < r) (x : ℝ):
     exponentialCdfReal r x = ∫ x in Iic x, exponentialPdfReal r x := by
-  rw [exponentialCdfReal,cdf_eq_toReal]
-  simp only [expMeasure, measurableSet_Iic, withDensity_apply]
-  rw [integral_eq_lintegral_of_nonneg_ae]; exact rfl
-  · exact ae_of_all _ fun a ↦ by simp [Pi.zero_apply, exponentialPdfReal_nonneg Fact.out a]
-  · exact (Measurable.aestronglyMeasurable (measurable_exponentialPdfReal r)).restrict
+  apply gammaCdfReal_eq_integral zero_lt_one hr
```

Comment:
Please use `exact` instead of `apply` when it closes the goal. But here you can simply delete the `by` at the end of the previous line and delete the `apply`.